src/main/java/core/model/Driver.java

```diff
@@ -34,12 +38,29 @@ public void setLicenceNumber(String licenceNumber) {
         this.licenceNumber = licenceNumber;
     }
 
+    public String getLogin() {
+        return login;
+    }
+
+    public void setLogin(String login) {
+        this.login = login;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
     @Override
     public String toString() {
         return "Driver{"
-                + "id=" + id
-                + ", name='" + name + '\''
-                + ", licenceNumber='" + licenceNumber + '\''
-                + '}';
+            + "id=" + id
+            + ", name='" + name + '\''
+            + ", licenceNumber='" + licenceNumber + '\''
+            + ", login='" + login + '\''
+            + '}';
```

Comment:
what if we'll get to compare two Driver object with different login/password? what will be the result?

Reply:
You mean, I also have to override equals and hashcode?

Reply:
Exactly